can-define.js

```diff
@@ -859,7 +859,13 @@ define.types = {
 	 * any string, including "", is truthy.
 	 */
 	'htmlbool': function(val) {
-		return typeof val === "string" || !!val;
+		if (val === '') {
+			return true;
+		}
+		if (val == null) {
```

Comment:
This null/undefined check is ensure no participation in https://github.com/canjs/canjs/issues/2316 (null/undefined should bypass type coercion).